Gemfile

```diff
@@ -26,6 +25,7 @@ group :test, :development do
   gem 'timecop', '~> 0.5'
   gem 'capybara'
   gem 'byebug'
+  gem 'pry'
```

Comment:
a trifling thing, but if you just amend the entry above this one to `pry-byebug` you'll get pry down as a dependency anyway.